pageserver/src/tenant_mgr.rs

```diff
@@ -257,6 +257,7 @@ pub fn activate_tenant(tenant_id: ZTenantId) -> anyhow::Result<()> {
                 false,
                 move || crate::tenant_threads::gc_loop(tenant_id),
             )
+            .map(|_| ())
```

Comment:
NIT: is it needed?

Reply:
One change is made in this PR to return `thread_id` upon spawning new thread. the `activate_tenant` requires the return signature to `Result<()>`, so this mapping is needed. Will add a comment to make this more clear

Reply:
Or you could give an omitted parameter a readable name?
```suggestion
            .map(|_thread_id | ())
```